test/general-use.js

```diff
@@ -33,17 +33,22 @@ module.exports = {
     english: {
 
         // english (appropriate mode) (sorted)
+        "Jack": "anga;quesse:a,tilde-below",
+        "happy": "hyarmen;parma:a,tilde-below;long-carrier:y",
```

Comment:
We will need to create mappings for y-above in `dan-smith.js`. The y in this notation currently refers to double dots below and can only be carried by the round carrier (which looks like a Latin "c").

https://tengwarjs-fpft2n6u3.now.sh/#y

We can change the names in this notation as long as we are through. It may be better to make "y" imply "above", or make both "y-above" and "y-below" explicit.

Reply:
Then for the Sindarin vowel Y, y-sindar?

Reply:
Can you describe the appearance of y-sindar? I’m only aware of two dots below and the inverted chapeau above.